src/components/look-controls.js

```diff
@@ -242,5 +274,13 @@ module.exports.Component = registerComponent('look-controls', {
 
   onTouchEnd: function () {
     this.touchStarted = false;
+  },
+
+  onPointerLockChange: function () {
+    this.pointerLocked = !!(document.pointerLockElement || document.mozPointerLockElement);
```

Comment:
shouldn't we check that `document.pointerLockElement`/`document.mozPointerLockElement` is `this.el.sceneEl`?